diff --git a/node_modules/@types/jest-environment-puppeteer/index.d.ts b/node_modules/@types/jest-environment-puppeteer/index.d.ts
index a0c720d..8b7de13 100644
--- a/node_modules/@types/jest-environment-puppeteer/index.d.ts
+++ b/node_modules/@types/jest-environment-puppeteer/index.d.ts
@@ -64,9 +64,8 @@ interface Global extends GlobalType.Global {
 }
 
 /** Note: TestEnvironment is sandboxed. Each test suite will trigger setup/teardown in their own TestEnvironment. */
-declare class PuppeteerEnvironment implements JestEnvironment {
+declare class PuppeteerEnvironment extends JestEnvironment {
   context: Context | null;
-  fakeTimers: FakeTimers<Timer> | null;
   global: Global;
   moduleMocker: ModuleMocker | null;
   constructor(config: Config.ProjectConfig);
