diff --git a/node_modules/@types/mkdirp-promise/index.d.ts b/node_modules/@types/mkdirp-promise/index.d.ts
index a4f389b..d4ed1b3 100644
--- a/node_modules/@types/mkdirp-promise/index.d.ts
+++ b/node_modules/@types/mkdirp-promise/index.d.ts
@@ -9,5 +9,5 @@ import mkdirp = require("mkdirp");
 declare function mkdirpPromise(
     path: string,
     opts?: mkdirp.Mode | mkdirp.Options
-): Promise<mkdirp.Made>;
+): Promise<mkdirp.Mode>;
 export = mkdirpPromise;
