diff --git a/node_modules/prettier-plugin-packagejson/lib/index.js b/node_modules/prettier-plugin-packagejson/lib/index.js
index ef1cdd9..41b4240 100644
--- a/node_modules/prettier-plugin-packagejson/lib/index.js
+++ b/node_modules/prettier-plugin-packagejson/lib/index.js
@@ -1,4 +1,4 @@
-const { parsers } = require('prettier/parser-babylon')
+const { parsers } = require('prettier/parser-babel')
 const sortPackageJson = require('sort-package-json')
 const parser = parsers['json-stringify']
 
